test/sequential/rewardsv2.poolLiquidityRewards.test.ts

```diff
@@ -213,6 +214,25 @@ describe("rewards v2 tests", () => {
         assetAmount.divn(2)
       );
     });
+
+    test("Given a user with Liquidity activated When tries to mint some more Then the user activated amount will grow on that value", async () => {
+      const mintingLiquidity = await mintLiquidity(
+        testUser3.keyRingPair,
+        MGA_ASSET_ID,
+        secondCurrency,
+        defaultCurrencyValue
+      );
+
+      const tokensReservedValue: BN[] = mintingLiquidity
+        .filter(
+          (item) =>
+            item.method === "LiquidityActivated" &&
+            item.section === "proofOfStake"
+        )
+        .map((x) => new BN(x.eventData[2].data.toString()));
+
+      expect(tokensReservedValue[0]).bnEqual(defaultCurrencyValue);
```

Comment:
These lines only validate the event. Can you validate user tokens storage?
After minting should be something like: before + tokensReservedValue[]